pkg/interface/package.json

```diff
@@ -10,7 +10,7 @@
     "@reach/tabs": "^0.10.5",
     "@tlon/indigo-dark": "^1.0.6",
     "@tlon/indigo-light": "^1.0.6",
-    "@tlon/indigo-react": "1.2.16",
+    "@tlon/indigo-react": "1.2.17",
```

Comment:
Surprised this works when 1.2.17 isn't released.

Reply:
Actually [I take that back](https://www.npmjs.com/package/@tlon/indigo-react), the GitHub and NPM have gotten out of sync.